examples/factors/README.md

```diff
@@ -26,101 +27,128 @@ cd wherever/you/want
 cargo risczero new factors
 ```
 This will create a project named `factors` in the directory where you ran the `cargo risczero new` command. Now we can enter our new project's directory and start working on it!
-```
+```sh
 cd factors
 ```
 
-## Step 2: Give the guest program a name
+## Step 2: Give the methods package a name
 
-This project will call a program that executes on the guest zkVM.
-It's currently named `method_name`.
-We want to name it something that represents what the guest program does — let's call it `multiply`.
-Edit `methods/guest/Cargo.toml`, changing the line `name = "method_name"` to instead read
+The methods package contains the program that executes on the guest zkVM as well as a few supporting libraries. Before we proceed, let's change its name in `methods/Cargo.toml` to match the name of our project, renaming `name = methods` to `name = factors-methods`.
+
+Parts of this package are included in the driver program `src/main.rs`, so change the line
+
+```no_compile
+use methods::{METHOD_NAME_ELF, METHOD_NAME_ID};
 ```
-name = "multiply"
+
```

Comment:
feels like we're missing a "to" here, but it's hard to parse in this diff format